multik-api/src/main/kotlin/org/jetbrains/kotlinx/multik/ndarray/data/NDArray.kt

```diff
@@ -256,7 +258,7 @@ public class NDArray<T, D : Dimension> constructor(
         if (this.dim.d == -1) throw Exception("Array dimension is undefined")
         if (this.dim.d > 4) return this as NDArray<T, DN>
 
-        return NDArray(this.data, this.offset, this.shape, this.strides, this.dtype, DN(this.dim.d))
+        return NDArray(this.data, this.offset, this.shape, this.strides, this.dtype, DN(this.dim.d), base = base ?: this)
```

Comment:
Should pass "base" in this function?

Reply:
I think so.
Newly returned array has the same MemoryView as a current one, so we should store the base array.

Reply:
But it also has the same shape, strides, offset; except dim